dockerfiles/Dockerfile-mina-test-executive

```diff
@@ -1,9 +1,9 @@
-ARG image=debian:stretch-slim
+ARG image=debian:bullseye-slim
```

Comment:
The next LTS after stretch is buster. Should it be buster instead of bullseye here?